app/controllers/idv/verify_info_controller.rb

```diff
@@ -1,12 +1,55 @@
 module Idv
   class VerifyInfoController < ApplicationController
+    include StringRedacter
+    include IdvSession
+
     before_action :render_404_if_verify_info_controller_disabled
     before_action :confirm_two_factor_authenticated
     before_action :confirm_ssn_step_complete
 
     def show
       increment_step_counts
       analytics.idv_doc_auth_verify_visited(**analytics_arguments)
+
+      redirect_to failure_url(:fail) and return if any_throttled?
+
+      process_async_state(load_async_state)
+    end
+
+    def update
+      return if idv_session.verify_info_step_document_capture_session_uuid
+
+      pii[:uuid_prefix] = ServiceProvider.find_by(issuer: sp_session[:issuer])&.app_id
+
+      if ssn_throttle.throttled_else_increment?
```

Comment:
I see the SSN rate limiting in effect here, but I do not see the resolution rate limiting in effect. I only see that in the new method. Is there something I am missing?

Reply:
There are two different throttles as you noted. In the old code they are both referred to as "throttle." We decided to resolve the confusion. Verify_base_step was only checking the ssn throttle on enqueue_job (see second link) which is our update method. Do you want us to add resolution throttle here? See https://github.com/18F/identity-idp/blob/cc482cb673277ddd993ac84daeade4384cdaf790/app/services/idv/steps/verify_base_step.rb#L72-L74 and https://github.com/18F/identity-idp/blob/cc482cb673277ddd993ac84daeade4384cdaf790/app/services/idv/steps/verify_base_step.rb#L175-L177

Reply:
Chatted with the team; we're going to address this in the PR for LG-7401